### ISSUE TITLE
Timeout Issue while connecting to a Call

### ISSUE THREAD

**Issue by adityagokula2210** [cometchat-pro/android-chat-sdk#30]
I try to call, user at the other end accept it n then the call ends automatically n im recieving this error in log
: [features/base/lib-jitsi-meet] Failed to load config from https://rtc-us.cometchat.io/config.js?room=1589886198a68b1c84efd5aae25e13ba9148eed9186e6b510c Error(Error){“message”:“timeout”}

For more information please check:

https://forum.cometchat.com/t/audio-and-video-calling-issue/1390

**Comment by adityagokula2210** [comment]
We were able to generate the issue at our end. However, the issue seems to be very random and was not generated every single time we tested. We have gone ahead and added a couple of failsafes for the Calling mechanism at our end. Also, we have added a retry logic if the timeout issue is encountered.
Now the SDK will try to connect to the call 3 times, if it encounters the timeout error.
We should be able to release a version of the Android SDK latest by tomorrow end of the day. 

Thanks
